assets/js/app.js

```diff
@@ -0,0 +1,26 @@
+// any CSS you require will output into a single css file (app.css in this case)
+import '../scss/app.scss';
+
+// require jQuery normally
+import $ from 'jquery';
+
+// create global $ and jQuery variables
+// eslint-disable-next-line
+global.$ = global.jQuery = $;
+
+import 'jquery.scrollto';
+import 'jquery-ui/ui/widget';
+import 'jquery-ui/ui/widgets/sortable';
+import 'bootstrap';
+import 'moment';
+import '../vendor/bootstrap-datetimepicker.min';
```

Comment:
Should we add a comment to explain why the import is 
```
import '../vendor/bootstrap-datetimepicker.min';
```
instead of something like
```
import 'bootstrap-datetimepicker';
```
?

(Maybe also for 
```
import 'x-editable/dist/bootstrap3-editable/js/bootstrap-editable';
import 'select2/dist/js/select2.full';
```

Reply:
Will do

Reply:
Done